documentation/plan_functions.md

```diff
@@ -12,7 +12,7 @@ Deep merges a hash of facts with the existing facts on a target.
 add_facts(Target $target, Hash $facts)
 ```
 
-*Returns:* `Variant[Target, Hash[String, Data]]` The target's new facts or a `Target` object if you are using the `future` flag
+*Returns:* `Variant[Target, Hash[String, Data]]` The target's new facts or a `Target` object if the `future` flag is set to `true`
```

Comment:
@cwcowellshah can you `git rm` this file. This is no longer managed in the bolt repo. I think the fix you reqested here is merged with https://github.com/puppetlabs/bolt/pull/1328 . Alternately you could rebase on master and resolve merge conflict by deleting plan_function.md